test/focusable-arrowkeys-behavior.test.js

```diff
@@ -1,27 +1,23 @@
 import '../d2l-dom-focus.js';
 import './focusable-arrowkeys-behavior-components.js';
-import { expect, fixture, html } from '@open-wc/testing';
-import { afterNextRender } from '@polymer/polymer/lib/utils/render-status.js';
-import { keyDown } from '@brightspace-ui/core/tools/dom-test-helpers.js';
+import { expect, fixture, html, oneEvent, sendKeysElem } from '@brightspace-ui/testing';
 
 describe('d2l-focusable-arrowkeys-behavior', () => {
 	let simpleFixture, focusables;
 
 	const testKeyInteractions = (keyInteractions) => {
 		keyInteractions.forEach((keyInteraction) => {
-			it(keyInteraction.name, (done) => {
+			it(keyInteraction.name, async() => {
 				simpleFixture.arrowKeyFocusablesOnBeforeFocus = (elem) => {
 					return new Promise((resolve) => {
 						expect(elem).to.equal(focusables[keyInteraction.endIndex]);
 						resolve();
 					});
 				};
-				focusables[keyInteraction.startIndex].focus();
-				focusables[keyInteraction.endIndex].addEventListener('focus', () => {
-					expect(D2L.Dom.Focus.getComposedActiveElement()).to.equal(focusables[keyInteraction.endIndex]);
-					done();
-				});
-				keyDown(focusables[keyInteraction.startIndex], keyInteraction.keyCode);
+				const e = oneEvent(focusables[keyInteraction.endIndex], 'focus');
+				sendKeysElem(focusables[keyInteraction.startIndex], 'press', keyInteraction.key);
+				await e;
```

Comment:
Was this happening too fast even without the `await` for this to work?
```
sendKeysElem(focusables[keyInteraction.startIndex], 'press', keyInteraction.key);
await oneEvent(focusables[keyInteraction.endIndex], 'focus');
```

Reply:
Hmm I definitely did try that, but some other stuff may have been broken at the time. Just tried it again and yes that works -- updated!